modular_citadel/code/modules/fish/fish_items.dm

```diff
@@ -0,0 +1,180 @@
+
+//////////////////////////////////////////////
+//			Aquarium Supplies				//
+//////////////////////////////////////////////
+
+/obj/item/weapon/egg_scoop
+	name = "fish egg scoop"
+	desc = "A small scoop to collect fish eggs with."
+	icon = 'modular_citadel/icons/obj/fish_items.dmi'
+	icon_state = "egg_scoop"
+	slot_flags = SLOT_BELT
+	throwforce = 0
+	w_class = WEIGHT_CLASS_SMALL
+	throw_speed = 3
+	throw_range = 7
+
+/obj/item/weapon/fish_net
+	name = "fish net"
+	desc = "A tiny net to capture fish with. It's a death sentence!"
+	icon = 'modular_citadel/icons/obj/fish_items.dmi'
+	icon_state = "net"
+	slot_flags = SLOT_BELT
+	throwforce = 0
+	w_class = WEIGHT_CLASS_SMALL
+	throw_speed = 3
+	throw_range = 7
+
+	suicide_act(mob/user)			//"A tiny net is a death sentence: it's a net and it's tiny!" https://www.youtube.com/watch?v=FCI9Y4VGCVw
```

Comment:
Use absolute pathing instead of relative pathing for procs

Reply:
Necessary for the way suicide_act is handled, literally any time this is called it's done as mob/user

Reply:
The proc itself is relatively pathed. It can be moved to absolute pathing just fine.

Using absolute pathing for procs makes it near impossible to actually search for the actual proc via github search or atom's search features, as relative pathing appears exactly the same as any random proc.